### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by Zolmeister**

### CODE REVIEW

File: index.dust [hunk 1]
```diff
@@ -54,7 +54,7 @@
     <link rel="privacy" href="http://clay.io/privacy">
   </head>
 
-  <body class="src-kik">
+  <body{?isProbablyKik} class="src-kik"{/isProbablyKik}>
```
Comment (austinhallock) [inline index.dust]: you can remove this. we're not using it anywhere

File: src/components/home/index.styl [hunk 2]
```diff
@@ -24,10 +24,16 @@ json('../../stylus/vars.json')
         font-size: $fontSizeMedium
         margin-bottom: 20px
 
-      > .google-play, .kik
+      > .kik
         height: 40px
         margin-bottom: 40px
 
+      > a
```
Comment (austinhallock) [inline src/components/home/index.styl]: why `a` and not `.google-play`?

File: test/server.coffee [hunk 3]
```diff
@@ -70,13 +74,39 @@ describe 'index.dust', ->
...
+        .flare (flare) ->
+          flare.res.body.should.contain \
+            '<title>Slime - Clay Games Mobile HTML5</title>'
+
+    it 'Responds with clay title if game not found', ->
+      flare
+        .get '/game/NOT_A_VALID_GAME_KEY'
+        .flare (flare) ->
+          flare.res.body.should.contain \
+            '<title>Clay Games - Play Free HTML5 Mobile Games</title>'
```
Comment (austinhallock) [inline test/server.coffee]: Hmm, Google will see this as a bunch of our pages having that title, which if I remember correctly is bad. Can we just 404 and redirect to homepage?

**Comment by Zolmeister** [comment]
updated

**Pull request merged**